### PR TITLE
Revert "Revert "Add translate="no" to tagging examples, add troubleshooting e…"

### PR DESCRIPTION
**Pull request by nosto-mikpou**
Reverts Nosto/wiki-docs#144

### CODE REVIEW

File: implementing-nosto/manual-implementation/order-tagging.md [hunk 1]
```diff
@@ -71,3 +71,19 @@ You can further verify your session in the Nosto admin by using the live feed un
 
 ![Nosto debug toolbar order](https://nosto-campaign-assets.s3.amazonaws.com/images/nosto-debug-toolbar-order.png) ![live-feed-product-order](https://nosto-campaign-assets.s3.amazonaws.com/images/live-feed-order.png)
 
+### Translate attribute
+
+The translate attribute is a [HTML5 standard attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/translate) which specifies whether the value of the element and it's `Text` node children should be translated. If your tagging elements are being translated by e.g. Google Translator then this is the way to opt out elements being translated by Google and possibly other vendors.
+
+```markup
```
Comment (timowestnosto) [inline implementing-nosto/manual-implementation/order-tagging.md]: is this example needed?

File: implementing-nosto/manual-implementation/category-and-brand-tagging.md [hunk 2]
```diff
@@ -45,3 +45,19 @@ Once included on all pages, you can review if the site is transmitting data usin
 
 ![Nosto debug category ](https://nosto-campaign-assets.s3.amazonaws.com/images/nosto-debug-toolbar-category.png)
 
+### Translate attribute
+
+The translate attribute is a [HTML5 standard attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/translate) which specifies whether the value of the element and it's `Text` node children should be translated. If your tagging elements are being translated by e.g. Google Translator then this is the way to opt out elements being translated by Google and possibly other vendors.
+
+```markup
```
Comment (timowestnosto) [inline implementing-nosto/manual-implementation/category-and-brand-tagging.md]: is this example needed?

Comment (nosto-mikpou) [reply implementing-nosto/manual-implementation/category-and-brand-tagging.md]: No, these were unsaved files

File: implementing-nosto/manual-implementation/order-tagging.md (hunk 1 above)
Comment (nosto-mikpou) [reply implementing-nosto/manual-implementation/order-tagging.md]: No, these were unsaved files

**Review by timowestnosto — APPROVED** [review]

**Pull request merged**